WalletWasabi.Fluent/Views/AddWalletPageView.axaml

```diff
@@ -0,0 +1,68 @@
+<UserControl xmlns="https://github.com/avaloniaui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:vm="clr-namespace:WalletWasabi.Fluent.ViewModels"
+             xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls"
+             xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters"
+             xmlns:i="clr-namespace:Avalonia.Xaml.Interactivity;assembly=Avalonia.Xaml.Interactivity"
+             xmlns:behaviors="clr-namespace:WalletWasabi.Fluent.Behaviors"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="vm:HomePageViewModel"
+             x:Class="WalletWasabi.Fluent.Views.AddWalletPageView">
+    <c:ContentArea Title="Add Wallet" Caption="Type in your wallets name and select an option:">
+        <StackPanel Spacing="40">
+            <StackPanel>
+                <Label Content="Wallet _Name:" Target="walletNameTb" />
+                <TextBox x:Name="walletNameTb" Text="{Binding WalletName}" Watermark="(e.g. My Wallet)">
+                    <i:Interaction.Behaviors>
+                        <behaviors:FocusOnAttachedBehavior />
+                    </i:Interaction.Behaviors>
+                </TextBox>
+            </StackPanel>
+            <StackPanel Spacing="10">
```

Comment:
I would add ScrollViewer here, when you reduce the window height not all buttons are visible

Reply:
done